storage/grpc_client.go

```diff
@@ -270,13 +275,56 @@ func (c *grpcStorageClient) OpenWriter(ctx context.Context, w *Writer, opts ...s
 // IAM methods.
 
 func (c *grpcStorageClient) GetIamPolicy(ctx context.Context, resource string, version int32, opts ...storageOption) (*iampb.Policy, error) {
-	return nil, errMethodNotSupported
+	// TODO: Need a way to set UserProject, potentially in X-Goog-User-Project system parameter.
```

Comment:
So every Storage v2 request message had a `CommonRequestParams` message field that contained a `string user_project` field. This is how it is set for every other RPC in the API. However, IAMPolicy methods use the google.iam.v1 request messages, so Storage doesn't own those and can't add the `CommonRequestParams`, so no way to set `user_project`.

There is an on going thread about if the Google System Parameter `X-Goog-User-Project` should be used instead of the request message field. The IAMPolicy methods would benefit from this, but even if that transition isn't made, I think the system parameter would need to be supported for IAMPolicy methods at the very least. I've raised this.